src/main/java/be/uantwerpen/minelabs/item/ItemGroups.java

```diff
@@ -44,21 +44,183 @@ public class ItemGroups {
+
+                stack.add(new ItemStack(Items.UPQUARK_RED));
+                stack.add(new ItemStack(Items.UPQUARK_GREEN));
+                stack.add(new ItemStack(Items.UPQUARK_BLUE));
+                stack.add(new ItemStack(Items.ANTI_UPQUARK_RED));
+                stack.add(new ItemStack(Items.ANTI_UPQUARK_GREEN));
+                stack.add(new ItemStack(Items.ANTI_UPQUARK_BLUE));
+                stack.add(new ItemStack(Items.DOWNQUARK_RED));
+                stack.add(new ItemStack(Items.DOWNQUARK_GREEN));
+                stack.add(new ItemStack(Items.DOWNQUARK_BLUE));
+                stack.add(new ItemStack(Items.ANTI_DOWNQUARK_RED));
+                stack.add(new ItemStack(Items.ANTI_DOWNQUARK_GREEN));
+                stack.add(new ItemStack(Items.ANTI_DOWNQUARK_BLUE));
+                stack.add(new ItemStack(Items.GLUON));
+                stack.add(new ItemStack(Items.ELECTRON));
+                stack.add(new ItemStack(Items.POSITRON));
+                stack.add(new ItemStack(Items.PHOTON));
+                stack.add(new ItemStack(Items.NEUTRINO));
+                stack.add(new ItemStack(Items.ANTINEUTRINO));
+                stack.add(new ItemStack(Items.WEAK_BOSON));
+                stack.add(new ItemStack(Items.PROTON));
+                stack.add(new ItemStack(Items.ANTI_PROTON));
+                stack.add(new ItemStack(Items.NEUTRON));
+                stack.add(new ItemStack(Items.ANTI_NEUTRON));
+                stack.add(new ItemStack(Items.PION_NUL));
+                stack.add(new ItemStack(Items.PION_MINUS));
+                stack.add(new ItemStack(Items.PION_PLUS));
+                stack.add(new ItemStack(Items.CHARGED_POINT));
+            })
             .build();
 
     public static final ItemGroup QUANTUM_FIELDS = FabricItemGroupBuilder.create(
                     new Identifier(Minelabs.MOD_ID, "quantum_fields"))
             .icon(() -> new ItemStack(Items.GLUON_QUANTUMFIELD))
+            .appendItems(stack ->{
+                stack.add(new ItemStack(Items.UPQUARK_QUANTUMFIELD));
+                stack.add(new ItemStack(Items.DOWNQUARK_QUANTUMFIELD));
+                stack.add(new ItemStack(Items.GLUON_QUANTUMFIELD));
+                stack.add(new ItemStack(Items.ELECTRON_QUANTUMFIELD));
+                stack.add(new ItemStack(Items.PHOTON_QUANTUMFIELD));
+                stack.add(new ItemStack(Items.NEUTRINO_QUANTUMFIELD));
+                stack.add(new ItemStack(Items.WEAK_BOSON_QUANTUMFIELD));
+                stack.add(new ItemStack(Items.UPQUARK_QUANTUMFIELD));
+                stack.add(new ItemStack(Items.DOWNQUARK_QUANTUMFIELD));
+                stack.add(new ItemStack(Items.GLUON_QUANTUMFIELD));
+                stack.add(new ItemStack(Items.ELECTRON_QUANTUMFIELD));
+                stack.add(new ItemStack(Items.PHOTON_QUANTUMFIELD));
+                stack.add(new ItemStack(Items.NEUTRINO_QUANTUMFIELD));
+                stack.add(new ItemStack(Items.WEAK_BOSON_QUANTUMFIELD));
```

Comment:
Duplicate entries.
Adding them once is enough.

Reply:
@pixar02 i know, I saw that someone did mologram that way.